src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTest.java

```diff
@@ -16,7 +17,7 @@
 
 public class JSONCodecRegistryTest {
 
-  private final JSONCodecRegistryTestData TEST_DATA = new JSONCodecRegistryTestData();
+  private static final JSONCodecRegistryTestData TEST_DATA = new JSONCodecRegistryTestData();
```

Comment:
as a pattern, i had been keeping the "TestData" class as instance but I guess given how tests work static makes sense.

Reply:
That is unfortunate, but it looks like ParameterizedTest requires source method (I tried and non-static ones do not seem to work? LMK if you know of a way) to be `static`.
I'd prefer non-static as well so if there's a way, that'd be better.